packages/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-requests.ts

```diff
@@ -0,0 +1,23 @@
+import * as z from "zod";
+import { baseRequestSchema, XCPDGatewaySchema } from "../shared";
+
+const patientDiscoveryDefaultSchema = baseRequestSchema.extend({
+  patientResource: z.any(),
```

Comment:
just for viz, I think z.any here is ok just because it would be too much of a pain in the ass to define a whole schema for the huge patient fhir resource

Reply:
Yea that was the idea. The empty object was giving me type erorrs